Cards/MadGraph5_aMCatNLO/NewTests/QCD-4Jets_HT-1000to1200_madgraphMLM-BGenFilter-pythia8/QCD-4Jets_HT-1000to1200_madgraphMLM-BGenFilter-pythia8.json

```diff
@@ -0,0 +1,18 @@
+{
+    "gridpack_submit": true,
+    "gridpack_path" : "QCD/QCD-4Jets_HT-1000to1200_madgraphMLM-pythia8",
+    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/madgraphMLM-hepmcfilter-pythia8.dat", "Filter/LHE_BGenFilterEQ0_filter.dat"],
```

Comment:
for the ordering let's do 'pythia8-hepmcfilter' since it happens after hepmc generation